packages/frontend/src/components/common/modal/DashboardExportModal.tsx

```diff
@@ -1,28 +1,70 @@
-import { type Dashboard } from '@lightdash/common';
+import { type Dashboard, type SchedulerFilterRule } from '@lightdash/common';
 import {
     Box,
     Button,
     Group,
     Modal,
+    SegmentedControl,
     Stack,
     Title,
+    Tooltip,
     type ModalProps,
 } from '@mantine/core';
-import { IconEyeCog, IconFileExport } from '@tabler/icons-react';
+import { IconCsv, IconEyeCog, IconFileExport } from '@tabler/icons-react';
 import { useCallback, useState, type FC } from 'react';
 import { useLocation } from 'react-router-dom';
 import { PreviewAndCustomizeScreenshot } from '../../../features/preview';
 import { CUSTOM_WIDTH_OPTIONS } from '../../../features/scheduler/constants';
-import { useExportDashboard } from '../../../hooks/dashboard/useDashboard';
+import {
+    useExportCsvDashboard,
+    useExportDashboard,
+} from '../../../hooks/dashboard/useDashboard';
 import MantineIcon from '../MantineIcon';
 
 type Props = {
     gridWidth: number;
     dashboard: Dashboard;
 };
 
-export const DashboardExportModal: FC<Props & ModalProps> = ({
-    opened,
+type CsvExportProps = {
+    dashboard: Dashboard;
+};
+
+const CsvExport: FC<CsvExportProps & ModalProps> = ({ dashboard, onClose }) => {
+    const exportCsvDashboardMutation = useExportCsvDashboard();
+    const dashboardFilters: SchedulerFilterRule[] | undefined = [];
```

Comment:
I still need to get the `current` dashboard filters to send it to the backend, in the image we use the location, but we can't use a plain string here, any ideas @joaoviana  ?